docs/API_walkthrough.md

```diff
@@ -0,0 +1,620 @@
+
+We will create the following entities:
+Beekeeper.jsonld
+```json
+{
+   "id":"urn:ngsi-ld:Beekeeper:01",
+   "type":"Beekeeper",
+   "name":{
+       "type":"Property",
+       "value":"Scalpa"
+   },
+   "@context":[
+      "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+   ]
+}
+```
+Apiary.jsonld
+```json
+{
+   "id":"urn:ngsi-ld:Apiary:01",
+   "type":"Apiary",
+   "name":{
+      "type":"Property",
+      "value":"ApiarySophia"
+   },
+   "location": {
+      "type": "GeoProperty",
+      "value": {
+         "type": "Point",
+         "coordinates": [
+            24.30623,
+            60.07966
+         ]
+      }
+   },
+   "@context":[
+      "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+   ]
+}
+```
+Sensor_temperature.jsonld
+```json
+{
+   "id": "urn:ngsi-ld:Sensor:02",
+   "type": "Sensor",
+   "deviceParameter":{
+         "type":"Property",
+         "value":"temperature"
+   },
+   "@context": [
+      "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+   ]
+}
+```
+Sensor_humidity.jsonld
+```json
+{
+   "id": "urn:ngsi-ld:Sensor:01",
+   "type": "Sensor",
+   "deviceParameter":{
+         "type":"Property",
+         "value":"humidity"
+   },
+   "@context": [
+      "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+   ]
+}
+```
+Beehive.jsonld
+```json
+{
+   "id": "urn:ngsi-ld:BeeHive:01",
+   "type": "BeeHive",
+   "belongs": {
+      "type": "Relationship",
+      "object": "urn:ngsi-ld:Apiary:01"
+   },
+   "managedBy": {
+     "type": "Relationship",
+     "object": "urn:ngsi-ld:Beekeeper:01"
+   },
+   "location": {
+      "type": "GeoProperty",
+      "value": {
+         "type": "Point",
+         "coordinates": [
+            24.30623,
+            60.07966
+         ]
+      }
+   },
+   "temperature": {
+      "type": "Property",
+      "value": 22.2,
+      "unitCode": "CEL",
+      "observedAt": "2019-10-26T21:32:52.98601Z",
+      "observedBy": {
+         "type": "Relationship",
+         "object": "urn:ngsi-ld:Sensor:01"
+      }
+   },
+   "humidity": {
+      "type": "Property",
+      "value": 60,
+      "unitCode": "P1",
+      "observedAt": "2019-10-26T21:32:52.98601Z",
+      "observedBy": {
+         "type": "Relationship",
+         "object": "urn:ngsi-ld:Sensor:02"
+      }
+   },
+   "@context": [
+      "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld"
+   ]
+}
+```
+
+## Queries
+
+* We start by creating the beekeeper, the apiary, the beehive and sensors:
+
+```shell
+http POST http://localhost:8080/ngsi-ld/v1/entities Content-Type:application/ld+json < Beekeeper.jsonld
```

Comment:
it would be nice to present them in such a way: https://httpie.io/docs#raw-and-complex-json (users could then just copy / paste the commands without having to create intermediate files with sample data)